docs/contributing/release_process.md

```diff
@@ -45,101 +93,126 @@ Use the following template for a clean changelog after the release:
 
 ### Output
 
 <!-- Changes to Black's terminal output and error messages -->
 
-### Packaging
-
-<!-- Changes to how Black is packaged, such as dependency requirements -->
+### _Blackd_
 
-### Parser
+<!-- Changes to blackd -->
 
-<!-- Changes to the parser or to version autodetection -->
+### Integrations
 
-### Performance
+<!-- For example, Docker, GitHub Actions, pre-commit, editors -->
 
-<!-- Changes that improve Black's performance. -->
+### Documentation
 
+<!-- Major changes to documentation and policies. Small docs changes
+     don't need a changelog entry. -->
 ```
 
 ## Release workflows
 
-All _Blacks_'s automation workflows use GitHub Actions. All workflows are therefore
-configured using `.yml` files in the `.github/workflows` directory of the _Black_
+All of _Black_'s release automation use [GitHub Actions]. All workflows are therefore
+configured using YAML files in the `.github/workflows` directory of the _Black_
 repository.
 
+They are triggered by the publication of a [GitHub Release].
+
 Below are descriptions of our release workflows.
 
-### Docker
+### Publish to PyPI
 
-This workflow uses the QEMU powered `buildx` feature of docker to upload a `arm64` and
-`amd64`/`x86_64` build of the official _Black_ docker image™.
+This is our main workhorse workflow. It builds a [sdist] and [wheels] to upload to PyPI
+where the vast majority of users will download Black from. It's divided into three job
+groups:
 
-- Currently this workflow uses an API Token associated with @cooperlees account
+#### sdist + pure wheel
 
-### pypi_upload
+This single job builds the sdist and pure Python wheel (ie. a wheel that only contains
+Python code) using [build]. These artifacts are general-purpose and can be used on
+basically any platform supported by Python.
 
-This workflow builds a Python
-[sdist](https://docs.python.org/3/distutils/sourcedist.html) and
-[wheel](https://pythonwheels.com) using the latest
-[setuptools](https://pypi.org/project/setuptools/) and
-[wheel](https://pypi.org/project/wheel/) modules.
+#### mypyc wheels (…)
 
-It will then use [twine](https://pypi.org/project/twine/) to upload both release formats
-to PyPI for general downloading of the _Black_ Python package. This is where
-[pip](https://pypi.org/project/pip/) looks by default.
+We use [mypyc] to compile _Black_ into a CPython C Extension for significantly improved
+performance. Wheels built with mypyc are platform and Python version specific.
+[Supported platforms are documented in the FAQ](labels/mypyc-support).
 
-- Currently this workflow uses an API token associated with @ambv's PyPI account
+These matrix jobs use [cibuildwheel] which handles the complicated task of building C
+extensions for many environments for us. Since building these wheels is slow, there are
+multiple mypyc wheels jobs (hence the term "matrix") that build for a specific platform
+(noted in the job name in parentheses).
 
-### Upload self-contained binaries
+...
```

Comment:
Same

Reply:
I was trying to add some separation between the following sentence and the previous section since the sdist / pure wheel job also uses twine to upload the distributions. 
![image](https://user-images.githubusercontent.com/63936253/187311079-ceb73efc-c45d-40d2-b494-4d1a5b9d4df6.png)

Happy to take suggestions for better ways of making this clearer.

Reply:
Resolved in https://github.com/psf/black/pull/3242/commits/82b3765cdc39a97569783c635d84ace758f928c6.